easybuild/tools/utilities.py

```diff
@@ -99,5 +102,9 @@ def import_available_modules(namespace):
                 mod_name = module.split(os.path.sep)[-1].split('.')[0]
                 modpath = '.'.join([namespace, mod_name])
                 _log.debug("importing module %s" % modpath)
-                modules.append(__import__(modpath, globals(), locals(), ['']))
+                try:
+                    mod = __import__(modpath, globals(), locals(), [''])
+                except ImportError as err:
```

Comment:
I'm trying to avoid that i) I get an ugly traceback, ii) that I have to use `except ImportError` all over the place. If this goes wrong, something is very wrong (and it's not an EasyBuild bug), and we're done, so we throw up an EasyBuildError that will be catched higher up (to avoid a traceback, and only print the error message and exit).